src/Common/Core/Impl/IO/Exports.cs

```diff
@@ -2,12 +2,8 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.ComponentModel.Composition;
-using Microsoft.Common.Core.IO;
-using Microsoft.VisualStudio.ProjectSystem.Utilities;
 
-namespace Microsoft.VisualStudio.R.Package.ProjectSystem {
-
-    [AppliesTo("RTools")]
+namespace Microsoft.Common.Core.IO {
```

Comment:
`Export` was separated from `FileSystem` implementation so that test assemblies (e.g. `Components.Tests`) could either use real `FileSystem` or replace it with fake. If you move it to `Common.Core`, this won't work anymore.